modules/ossm-rn-new-features.adoc

```diff
@@ -15,6 +15,41 @@ Module included in the following assemblies:
 
 This release adds improvements related to the following components and concepts.
 
+== New features {SMProductName} version 2.4.3
+
+This release of {SMProductName} makes {SMProductName} operator to ARM based clusters available as a Technology Preview feature, adds the `envoyExtAuthzGrpc` field to configure an external authorization provider using the gRPC API, addresses Common Vulnerabilities and Exposures (CVEs), and is supported on {product-title} 4.10 and later versions.
+
+=== Component versions included in {SMProductName} version 2.4.3
+//THESE MAY NEED TO BE UPDATED FOR 2.4.3
+//Kiali updated to 1.65.8 on 09/06/2023
+//09/06/2023: According to distributed tracing release notes, Jaeger component version remains unchanged.
+|===
+|Component |Version
+
+|Istio
+|1.16.7
+
+|Envoy Proxy
+|1.24.10
+
+|Jaeger
+|1.42.0
+
+|Kiali
+|1.65.8
+|===
+//COMPONENTS ABOVE MAY NEED TO BE UPDATED FOR 2.4.3
+
+=== {SMProductName} operator to ARM based clusters
+:FeatureName: {SMProductName} operator to ARM based clusters
+include::snippets/technology-preview.adoc[]
+
+This release makes {SMProductName} operator to ARM based clusters available as a Technology Preview feature. Images are available for Istio, Envoy, Prometheus, Kiali, and Grafana. Images are not available for Jaeger so Jaeger must be disabled as a {SMProductShortName} addon.
+
+=== Remote Procedure Calls (gRPC) API support for external authorization configuration
+
+Remote Procedure Calls (gRPC) was only available as a Technical Preview feature. It was made generally available with 2.4.0. This enhancement adds the `envoyExtAuthzGrpc` field to configure an external authorization provider using the gRPC API.
```

Comment:
```suggestion
This enhancement adds the `envoyExtAuthzGrpc` field to configure an external authorization provider using the gRPC API.
```

I'd remove the first sentence - presumably it was already covered in release notes when gRPC went GA in the 2.4.0 release, so it doesn't seem that relevant here, but I could be wrong.